sources/osg/CullVisitor.js

```diff
@@ -50,6 +50,9 @@ var CullVisitor = function () {
     this._reserveLeafStack = [ new RenderLeaf() ];
     this._reserveLeafStack.current = 0;
 
+    this._reserveCullSettingsStack = [ new CullSettings() ];
```

Comment:
we can make a "generic" pool object, just have to "standardize" the init/clear/reset so that we can make "stack" for any object (leaf, cullsettings, etc.) ?